sockeye/inference.py

```diff
@@ -833,7 +836,14 @@ def translate(self, trans_inputs: List[TranslatorInput], fill_up_batches: bool =
                 batch = batch + [batch[0]] * rest
 
             translator_inputs = [indexed_translator_input.translator_input for indexed_translator_input in batch]
-            batch_translations = self._translate_nd(*self._get_inference_input(translator_inputs))
+
+            if self.mc_dropout:
```

Comment:
not a big fan of having this in the `Translator` class. Is there a way to keep this local to the model class somehow?

Reply:
I found a way to move it there, although split into two functions.

Reply:
Looks much better, thanks! Could you add a logging message in `load_model()` when `mc_dropout` is enabled? This would help debugging if users accidentally enable it and wonder about nondeterministic outputs :)